src/assets/sass/views/_staff.scss

```diff
@@ -58,3 +58,7 @@ p {
     background-color: white;
     height: 500px;
 }
+
+p {
```

Comment:
I'm curious if we should be adding styling to the <p> tag here instead of just creating a regular class and adding that.  I believe this will override the font-size for `<p>` everywhere else too.  But if we like that size for all paragraph text we can always move that to a better spot.  Alternatively, we can use CSS module syntax here and scope it to the component we want it in.

We can leave it here for now though, this is low-priority as long as it doesn't break existing pages.